api/Models/Response/AccountStageResults.cs

```diff
@@ -0,0 +1,5 @@
+using SpoRE.Infrastructure.Database;
+
+namespace SpoRE.Models.Response;
+
+public record AccountStageResults(Account account, int? stagescore, int? totalscore);
```

Comment:
niet nullable? Dan is default value 0 ipv null
en deze velden zijn eigenlijk helemaal niet nullable volgens mij.
maar de c# modellen kloppen niet altijd maar ik heb die pas geupdate als ik ze tegen kwam
![image](https://user-images.githubusercontent.com/15751288/236667218-51a2baa8-90b2-455b-ad8f-6780e4409412.png)
indd default value 0